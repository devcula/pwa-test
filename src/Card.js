import React from 'react';

const Card = () => {
    return(
        <div className="bg-light-green br3 ma2 grow dib pa3 bw2 shadow-5">
            <img src="https://robohash.org/devcula?100*100" alt="robo"/>
            <div className="tc">
                <h2>Jane Doe</h2>
                <p>jane.doe@gmail.com</p>
            </div>
        </div>
    )
}

export default Card; 