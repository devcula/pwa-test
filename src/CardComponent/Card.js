import React from 'react';

const Card = ({ id, name, email}) => {
    //const { id, name, email} = props;      //Used Destructuring directly above
    return(
        <div className="tc bg-light-green br3 ma2 grow dib pa3 bw2 shadow-5">
            <img src={`https://robohash.org/${id}?200x200`} alt="robo"/>
            <div>
                <h2>{name}</h2>
                <p>{email}</p>
            </div>
        </div>
    )
}

export default Card; 