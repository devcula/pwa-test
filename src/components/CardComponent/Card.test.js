import { shallow } from 'enzyme';
import Card from './Card';
import React from 'react';
import toJson from 'enzyme-to-json';

it("expect to render Card component", () => {
    const wrapper = shallow(<Card />);
    expect(toJson(wrapper)).toMatchSnapshot();
    expect(toJson(wrapper).children.length).toEqual(2);
})