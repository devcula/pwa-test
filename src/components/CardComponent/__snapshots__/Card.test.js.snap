// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<div
  className="tc bg-light-green br3 ma2 grow dib pa3 bw2 shadow-5"
>
  <img
    alt="robo"
    src="https://robohash.org/undefined?size=200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
